site2/docs/io-debezium-source.md

```diff
@@ -458,6 +458,58 @@ This example shows how to change the data of a MongoDB table using the Pulsar De
     ```
    
+## Example of Oracle
+
+### Packaging
+
+Oracle connector does not include Oracle JDBC driver and the end user will need to package it with the connector.
+Major reasons for not including the drivers are the variety of versions and Oracle licensing. It is recommended using the driver provided with your Oracle DB installation, or you can [download](https://www.oracle.com/database/technologies/appdev/jdbc.html) one.
+Integration test have an [example](https://github.com/apache/pulsar/blob/e2bc52d40450fa00af258c4432a5b71d50a5c6e0/tests/docker-images/latest-version-image/Dockerfile#L110-L122) of packaging the driver into the connector nar file. 
+
+### Configuration
+
+Debezium [requires](https://debezium.io/documentation/reference/1.5/connectors/oracle.html#oracle-overview) Oracle DB with LogMiner or XStream API enabled. 
+Supported options and steps for enabling them vary from version to version of Oracle DB.
+Steps outlined in the [documentation](https://debezium.io/documentation/reference/1.5/connectors/oracle.html#oracle-overview) and used in the [integration test](https://github.com/apache/pulsar/blob/master/tests/integration/src/test/java/org/apache/pulsar/tests/integration/io/sources/debezium/DebeziumOracleDbSourceTester.java) may or may not work for the version and edition of Oracle DB you are using.
+Please refer to the documentation for Oracle DB as needed.
+
+Similarly to other connectors, you can use JSON or YAMl to configure the connector.
```

Comment:
Like other connector docs, can you give a JSON example?

Reply:
done